#19238 |  Meta descriptions for dotnet wpf (#19238)
dotnet/docs: docs/framework/wpf/data/how-to-specify-the-binding-source.md

---
title: "How to: Specify the Binding Source"

> description: Find out about ways to specify the binding source through this example in the Windows Presentation Foundation (WPF).

Reviewer: I think this is a bit clearer.
```suggestion
description: Learn how to specify the binding source through this example in the Windows Presentation Foundation (WPF).
```
